src/routes/root.css

```diff
@@ -1,11 +1,107 @@
+/*==================
+  App Heading
+==================*/
+/*link to open external page and having icon after it*/
+.header a.external {
+  color: var(--goa-color-interactive-default);
+}
+.header a.external::after {
```

Comment:
Since this style is specific to the `.header` I think it is only the Report a bug link that will be affected. If this is the case using a regex will work once the existing url is pointed to github.